engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/ExecCompilerTest.java

```diff
@@ -528,7 +531,8 @@ public void illegalPrivateConversion() {
     var expectedErrMsg = DeclaredAsPrivate$.MODULE$.explain();
     var runMethod = module.invokeMember(Module.EVAL_EXPRESSION, "run");
     runMethod.execute(0);
-    assertThat(ctxRule.getOut(), containsString(expectedErrMsg));
```

Comment:
- @Akirathan was a great fan of `assertThat`
- and I started to use it as well
- can we keep `assertThat` when already present?